nucypher/cli/commands/worklock.py

```diff
@@ -264,20 +266,84 @@ def refund(general_config, worklock_options, registry_options, force, hw_wallet)
 
 
 @worklock.command()
+@group_general_config
+@group_registry_options
+@group_worklock_options
+@option_force
+@option_hw_wallet
+@click.option('--gas-limit', help="Gas limit per each verification transaction", type=click.IntRange(min=60000))
+# TODO: Consider moving to administrator (nucypher-deploy) #1758
+def post_init(general_config, registry_options, worklock_options, force, hw_wallet, gas_limit):
```

Comment:
Perhaps `def enable_claims`?

Reply:
I guess in this version of updates we can use this name, because no more `enable_claiming` function in contract

Reply:
+1!

Reply:
✔️